connectors/stash/src/test/resources/stash/responses/success.json

```diff
@@ -0,0 +1,171 @@
+{
+  "cards": [
+    {
+      "id": "00000000-0000-0000-0000-000000000000",
+      "creation_date": "1970-01-01T00:00:00Z",
+      "header": {
+        "title": "Stash PR - Feature/APF-651 add image and test auth test/Sudharsan Sathiamoorthy",
+        "subtitle": "Description: * APF-651-Added integration test for getImage and getAuthKey\r\n* APF-651-Added integration test for test auth for connectors"
+      },
+      "body": {
+        "description": "* APF-651-Added integration test for getImage and getAuthKey\r\n* APF-651-Added integration test for test auth for connectors",
+        "fields": [
+          {
+            "type": "GENERAL",
+            "title": "Repository: ",
```

Comment:
By the way, having the ": " in this string is causing the client to display things like: `Repository: :  app-platform-server`.

Don't specify the ": " in the text.properties file.